str/runners/str_iterative_eh_runner.py

```diff
@@ -85,8 +85,13 @@ def main(
     crams_by_id = {}
     for i in response['project']['sequencingGroups']:
         for cram in i['analyses']:
-            # ignore archived CRAMs
-            if 'archive' in cram['output']:
+            # ignore archived CRAMs and ONT crams
+            try:
+                if 'archive' in cram['output']:
+                    continue
+                elif 'ont' in cram['output']:
+                    continue
+            except: 
```

Comment:
Naked exceptions are not advised as they'll hide any error (including the ones you aren't expecting)

You could replace this try;except with `if not cram['output']: continue`, or `if cram['output'] is None: continue`